### PR TITLE
fix gm-control-api / redis race condition

### PR DESCRIPTION
**Pull request by dgoldstein1**
Adds in:

- https://github.com/greymatter-io/helm-charts/pull/851

### CODE REVIEW

**Review by zoemccormick — APPROVED** [review]
Can't quite tell why this is failing in the lint and test charts - but when that's resolved and an SRE signs of on merging to `release-2.4` this looks good to me!

**Comment by dgoldstein1** [comment]
I think this is an issue with `release-2.4`. I created https://github.com/greymatter-io/helm-charts/issues/880 to track this

@moosecanswim can you review this? It should be update to date with `release-2.4`

File: .chart-testing/fabric-sense-data.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 charts:
```
Comment (moosecanswim) [inline .chart-testing/fabric-sense-data.yaml:1]: Could you change this file to read:
```
target-branch: release-2.4
chart-dirs:
  - .
```

To test this locally you can run ` ct lint --config .chart-testing/fabric-sense-data.yaml `

this is the ct [github](https://github.com/helm/chart-testing)

**Review by moosecanswim — CHANGES_REQUESTED** [review]
there is an issue with the umberella chart testing action config which is in `.chart-testing/fabric-sense-data.yaml`

File: .chart-testing/fabric-sense-data.yaml (hunk 1 above)
Comment (chrisbsmith) [reply .chart-testing/fabric-sense-data.yaml:1]: Actually @dgoldstein1 , can you please make sure you rebase against `release-2.4` first? This file already exists in that branch with the changes @moosecanswim call out above . https://github.com/greymatter-io/helm-charts/blob/release-2.4/.chart-testing/fabric-sense-data.yaml

**Comment by moosecanswim** [comment]
@dgoldstein1 that change is in release-2.4 so you might just want to rebase onto it

**Comment by moosecanswim** [comment]
@dgoldstein1 i also just pushed an update to fabric and control-api charts so the version will need to be incremented again.

File: .chart-testing/fabric-sense-data.yaml (hunk 1 above)
Comment (dgoldstein1) [reply .chart-testing/fabric-sense-data.yaml:1]: I get the same thing on release-2.4
```
 david@Davids-MacBook-Pro  ~/dev/work/helm-charts   release-2.4  ct lint --config .chart-testing/fabric-sense-data.yaml


Linting charts...
Version increment checking disabled.
------------------------------------------------------------------------------------------------------------------------
No chart changes detected.
------------------------------------------------------------------------------------------------------------------------
Error: Error linting charts: Could not read 'Chart.yaml': open Chart.yaml: no such file or directory
Error linting charts: Could not read 'Chart.yaml': open Chart.yaml: no such file or directory
 ✘ david@Davids-MacBook-Pro  ~/dev/work/helm-charts   release-2.4  echo $?
1
```

**Comment by dgoldstein1** [comment]
closing in favor of https://github.com/greymatter-io/helm-charts/pull/883. After rebasing needed to push a new branch which requires a new PR.

**Comment by chrisbsmith** [comment]
`git push -f` 😄

**Pull request closed without merging**